src/main/java/appeng/parts/automation/PartImportBus.java

```diff
@@ -284,6 +285,31 @@ private boolean importStuff( InventoryAdaptor myAdaptor, IAEItemStack whatToImpo
 		return false;
 	}
 
+	private int calculateMaximumAmountToImport( InventoryAdaptor myAdaptor, IAEItemStack whatToImport, IMEMonitor<IAEItemStack> inv, FuzzyMode fzMode )
+	{
+		final int toSend = Math.min( this.itemToSend, 64 );
```

Comment:
It is still working with normal itemstacks, not their AE2 version. So 64 is the maximum for them.
Also I would not be surprised, if some inventories could crash if you ask to import a stack with a size of 96.